src/Umbraco.Core/Persistence/Repositories/Implement/DocumentRepository.cs

```diff
@@ -405,7 +405,7 @@ protected override void PersistNewItem(IContent entity)
             else if (content.PublishedState == PublishedState.Unpublishing)
             {
                 content.Published = false;
-                content.PublishTemplate = null;
+                content.PublishTemplateId = 0;
```

Comment:
Given that it was a null and we're using a Nullable<int> shouldn't we keep it as null